hmmlearn/utils.py

```diff
@@ -60,3 +60,9 @@ def iter_from_X_lengths(X, lengths):
 
         for i in range(len(lengths)):
             yield start[i], end[i]
+
+def prob_log(prob):
```

Comment:
Yes, you're right. Ignore my comment above :)

Do you have any ideas for a more descriptive name? I would suggest the *mask* thing, but I'm not sure if it's appropriate.

Could you add a comment (or even a proper docstring) referencing the original issue and explaining the motivation for this function?

One final thought. There is context manager [`np.errstate`](http://docs.scipy.org/doc/numpy-1.10.1/reference/generated/numpy.errstate.html#numpy.errstate) for turning warning on and off in a limited scope. You could use it instead of `np.seterr`.